starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +15,101 @@ var words = [
   'crackpot'
 ];
 
+function longest(words) {
```

Comment:
your function seems to be a little off. It should look something more like this 
`
function longest(words) {
var longest = "";
 for(i=0;i < words.length;i++) {
   if(words[i].length > longest.length) {
     longest = words[i];
     } else {
       continue;
     }
   }
 return longest;
}
`